content/features/featuresDeepDive/physics/v2/compounds.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Physics compounds

```

Comment:
I was mostly using "compounds" because it's the word used for V1, but the V2 equivalent is Container so I agree it sounds a bit strange... @CedricGuillemet I wonder if it's best to just use Container in this case?

Reply:
I was mostly using "compounds" because it's the word used for V1, but the V2 equivalent is Container so I agree it sounds a bit strange... @CedricGuillemet I wonder if it's best to just use Container in this case?